src/onegov/org/views/directory.py

```diff
@@ -57,6 +57,9 @@ def get_directory_form_class(
     model: object,
     request: 'OrgRequest'
 ) -> type[DirectoryForm]:
+    if 'faq' in request.path_info:
```

Comment:
This is very fragile, if we collapse the two forms into one we can get rid of this.

Reply:
I tried hard to implement the field dependencies but I didn't really manage :-( 
1) It seems the `DirectoryFAQForm` will be shown for regular directories as well as for faq directories
2) I didn't find a good way to change the form values in case of faq directories

Reply:
1. If you don't actually make the form dynamic, i.e. you have separate views it might be better to just use `on_request` and delete all the fields/fieldsets that should be hidden. Also don't use a field for `is_faq` if you go that route, just set/read it in the view. I personally think it would be better to just keep things simple and keep doing it all in one view. We don't gain that much from pre-populating two fields with a suggestion and if we really want to have that, we could still do it with JavaScript.
2. Don't change form values if you can avoid it, you can dynamically set a default, but I'd avoid setting data, it gets messy and is not really intended by WTForms.
3. Seems like a bug, although it will be a bit tricky to fix, since we will have to check in JavaScript whether the corresponding fieldset is empty after hiding a field and then hide the section. And do the reverse when a field is unhidden.

Reply:
Regarding point 1: I would prefer to have one view only but not sure how to accomplish this to distinguish e.g. 'new directory' from 'new faq directory'

Reply:
My question is, do we even need to make this as user-friendly? The ones setting up new directories are usually us. I'd honestly keep this as dead-simple as possible and add a checkbox "Allow anonymous submissions" which turns off the requirement of having an E-Mail field for user submissions.

As long as it's easy to submit new questions and add answers for those questions that's honestly good enough in my book. There may be other use cases where anonymous submissions make sense.

Reply:
And the display of individual entries as an accordion can be another checkbox.

Reply:
Or better yet just a generic radio select called `layout` with the two options, in case we want to add more layouts in the future.

Reply:
> My question is, do we even need to make this as user-friendly? The ones setting up new directories are usually us and this only needs to be done once per tenant. I'd honestly keep this as dead-simple as possible and add a checkbox "Allow anonymous submissions" which turns off the requirement of having an E-Mail field for user submissions.
> 
> As long as it's easy to submit new questions and add answers for those questions that's honestly good enough in my book. There may be other use cases where anonymous submissions make sense.

The goal is to have Q&A lists provided by the community. Personally I would not allow anyone submitting a new question, I would rather collect the most frequent one and publish them...

Reply:
The way I read the original issue the only thing they really care about is having a different layout anyways. I misread some of your code, that's where the anonymous submissions bit came from.

Reply:
I'd pretty much revert all of the changes to the form and views, the only thing I'd keep is the changes to the template and SCSS, and change the condition for the accordion layout to `directory.layout == 'accordion'` and add one radio field for layout to the directory form, I don't think we need any other changes. It will be much easier to maintain.

Point in case: the changes were so complicated I couldn't really figure out what the goal of the change was without reading the original ticket.

You also haven't ported the changes to the Org Template and SCSS yet, so the setting would not do anything in Org, which I'd give a higher priority than making it easy to configure.

Reply:
Right, not yet ported to org. 
My goal was to prepare the `form.structure` so anybody could start an Q&A catalog/directory ;-) But let me try your suggested layout setting.